src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.html

```diff
@@ -102,48 +119,48 @@ <h4 translate="KIBANA-ALERT_INDEX_PATTERN_DEPRECATED"></h4>
             </label>
             <select
               required
-              ng-options="opt.display for opt in index.nameIntervalOptions"
-              ng-model="index.nameInterval"
+              ng-options="opt.display for opt in ctrl.nameIntervalOptions"
```

Comment:
I see this `ctrl.` all over the code now, which I thought was going to be an alias for controller, and I was going to make a comment about how that's a poor way to name controllers. But that's not the case.

It actually appears to be some kind of state object on scope. This the naming is both generic and implies that it's something it's not. I'd like to see a more readable/understandable naming here. At least before when it was `index.`, you could assume it was properties of the index.